package.json

```diff
@@ -20,8 +20,8 @@
     "flat": "^5.0.0",
     "html-entities": "^1.2.1",
     "http-proxy": "^1.18.0",
-    "jose": "^1.10.1",
```

Comment:
Closes #37

Reply:
The pm2 deployment breaks with the new major version of convict. Might need to leave this for a future PR if there is some functionality in v6 that is desired.